### ISSUE TITLE
[OS X] [Linux] Screen corruption on loading screen

### ISSUE THREAD

**Issue by NeutralKaon** [tton-bugs-hunters/tton-bug-tracker#19]
As per [my forum post](https://forums.inxile-entertainment.com/viewtopic.php?f=47&t=17400&p=182797#p182797), there is OpenGL screen corruption on the loading screen reported on both OS X (10.11.6,  rMBP-15" Late 2013, GeForce 750M) and on Debian Sid with the Mesa amdgpu driver. This approximately manifests as shown below. 

![Example](https://s30.postimg.org/692gjdcc1/Screen_Shot_2017_03_02_at_09_06_56.png)

**Comment by shmerl** [comment]
Please post your OpenGL version.

**Comment by NeutralKaon** [comment]
Sure: 

`$ glxinfo | grep "OpenGL version
OpenGL version string: 2.1 NVIDIA-10.10.14 310.42.25f02`

**Comment by shmerl** [comment]
Well, at least in this case, it's not Mesa and amdgpu but Nvidia and its closed driver.

**Comment by NeutralKaon** [comment]
Yep -- the comment about mesa was from a 'me too' forum post. For what it is worth I have CUDA Driver Version: 8.0.46 installed (which I use for scientific computing).

**Comment by mafemergency** [comment]
If you draw on the upper or lower area of the screen it will not get cleared on
the next frame. I'd hazard a guess that the majority of the screen is scissor'd
for the video and then neglecting the non-scissor'd region, which might not have
zero'd out memory.

Somewhat related: https://www.khronos.org/opengl/wiki/Scissor_Test_And_Framebuffer_Clearing
